packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -16,6 +17,11 @@ import { inputFieldShared } from '@vaadin/field-base/src/styles/input-field-shar
 import { css, registerStyles, ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 const multiSelectComboBox = css`
+  :host {
+    --vaadin-multi-select-combo-box-chip-min-width: 60px;
+    --vaadin-multi-select-combo-box-input-min-width: 4em;
```

Comment:
I suggested this in the context of another issue (Tree Grid toggle level offset), and I feel this might be a suitable place for this approach. With the current implementation, it’s not possible to override these globally without targeting `vaadin-multi-select-combo-box` explicitly.

Also, I suggest we use `em` for the chip min width as well.

```suggestion
    --chip-min-width: var(--vaadin-multi-select-combo-box-chip-min-width, 4em);
    --input-min-width: var(--vaadin-multi-select-combo-box-input-min-width, 4em);
```

If we would do it like this, then the references to `var(--vaadin-multi-select-combo-box-chip-min-width)` and `var(--vaadin-multi-select-combo-box-input-min-width)` would need to be updated to the local/host-scoped variables instead.

Reply:
Updated. Also changed the overflow detection logic to get computed value of `--chip-min-width`.